### ISSUE TITLE
Invalid config error

### ISSUE THREAD

**Issue by piersmana** [firebase/firebaseui-web#324]
I am getting the error:

`
[firebaseui] Invalid config: "autoUpgradeAnonymousUsers"
`

From this config:

```
{
    'autoUpgradeAnonymousUsers': true,
    callbacks: {
        signInSuccess: function(currentUser, credential, redirectUrl) {
            return true;
        },
        signInFailure: function(error) {
            if (error.code != 'firebaseui/anonymous-upgrade-merge-conflict') {
                return Promise.resolve();
            }
            let cred = error.credential;
            // If using Firebase Realtime Database. The anonymous user data has to be
            // copied to the non-anonymous user.
            let app = firebase.app();
            // Save anonymous user data first.
            return app.database().ref(firebase.auth().currentUser.uid)
            .once('value')
            .then(function (snapshot) {
                data = snapshot.val();
                // This will trigger onAuthStateChanged listener which
                // could trigger a redirect to another page.
                // Ensure the upgrade flow is not interrupted by that callback
                // and that this is given enough time to complete before
                // redirection.
                return firebase.auth().signInWithCredential(cred);
            })
            .then(function (user) {
                // Original Anonymous Auth instance now has the new user.
                return app.database().ref(user.uid).set(data);
            })
            .then(function () {
                return anonymousUser.delete();
            }).then(function () {
                data = null;
                // FirebaseUI will reset and the UI cleared when this promise
                // resolves.
                // signInSuccess will not run. Successful sign-in logic has to be
                // run explicitly.
                window.location.assign('https://***');
            });
        }
    },
    signInFlow: 'redirect',
    signInSuccessUrl: 'https://***',
    signInOptions: [{
        provider: firebase.auth.GoogleAuthProvider.PROVIDER_ID,
        authMethod: 'https://accounts.google.com',
        clientId: '***.apps.googleusercontent.com'
    }],
    tosUrl: 'http://***/tos'
};
```

Checking the source I can only see this line creating this error:
[https://github.com/firebase/firebaseui-web/blob/d355eec36b97d337548e5d84e590f2cddada7dcf/javascript/widgets/config.js#L714](url)

What is happening here?

**Comment by wti806** [comment]
Which firebaseUI version are you using?

**Comment by piersmana** [comment]
2.5.1

**Comment by wti806** [comment]
Please upgrade to 2.6.1. The feature is only available starting from 2.6.0 But please use 2.6.1 since 2.6.0 has some minor issues which are fixed in 2.6.1

**Comment by piersmana** [comment]
Gotcha, thanks - I didn't see that version. That fixed it!
